subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3191,6 +3191,7 @@ static inline void event_len_prep(struct ll_conn *conn)
 #else /* !CONFIG_BT_CTLR_PHY */
 			tx_time = PKT_US(conn->default_tx_octets, 0);
 #endif /* !CONFIG_BT_CTLR_PHY */
+			conn->default_tx_time = tx_time;
```

Comment:
Move lines 3170-3172. to after line 3222 and have it as
```
#if defined(CONFIG_BT_CTLR_PHY)
		conn->default_tx_time = tx_time;
#endif /* CONFIG_BT_CTLR_PHY */
```

Use at line 3194, 3199, and 3208:
```
tx_time = conn->llcp_length.tx_time;
```

Reply:
except we must set conn->default_tx_time always, not only when we have set CONFIG_BT_CTLR_PHY

Reply:
Correct me, if CONFIG_BT_CTLR_PHY is not selected, then we should not use conn->default_tx_time it shall be compiled-out to save RAM space. refer to ull_conn_types.h

Reply:
FYI: the current master (without this PR) does not compile on our platform, because of potential uninitialized rx_time and tx_time. Please address this as well.

Reply:
I will address both issues